src/psyclone/tests/domain/lfric/transformations/dynamo0p3_transformations_test.py

```diff
@@ -6352,7 +6352,7 @@ def test_accenterdata_builtin(tmpdir):
         _, _ = acc_loop_trans.apply(loop)
     _, _ = parallel_trans.apply(sched.children)
     _, _ = acc_enter_trans.apply(sched)
-    output = str(psy.gen)
+    output = str(psy.gen).lower()
```

Comment:
Was there any reason for lowering the output in this test?

Reply:
Yes - in increasing the code-reuse the new code uses a different case to the old code (for directives) so I just opted to convert everything to lowercase in the test.